spec/rollbar/plugins/active_job_spec.rb

```diff
@@ -9,6 +9,11 @@ class TestJob
     include Rollbar::ActiveJob
 
     attr_reader :job_id
+    attr_accessor :arguments
+
+    def initialize(*arguments)
```

Comment:
@PlugIN73 I guess with this you are giving the same behavior than for a real ActiveJob job. I have a doubt, I think `arguments` will return the arguments for the `perform` method, no?

Can you make this test more close to a real scenario?